src/sass/index.scss

```diff
@@ -99,8 +101,10 @@ button{
 
 .navbar-default{
 	 border: none;
-	 padding: 12px;
    .navbar-nav li{
+		 a{
```

Comment:
You might want to look into spacing in this css file, It looks like 8 spaces, but 2 or 4 spaces is okay